README.md

```diff
@@ -1 +1,197 @@
+
+The DDD Familiarity Assessment goal is to provide you with a set of questions to assess the DDD knowledge within your organization.
+
+:warning: DDD Familiarity Assessment __does not__ intend to be a Maturity Model, and it should not be used as such.
+
+Instead, we expect to help organizations identify DDD knowledge gaps and use the results to prepare learnings, training, or similar to fill those gaps.
+
+## Familiarity Assessment Survey
+
+TODO
+
+- How do you assess yourself on DDD? (choose one)
+  - ( ) Never used
+  - ( ) I'm familiar with the concepts
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- How do you assess yourself on Strategic Design of DDD? (choose one)
+  - ( ) Never used
+  - ( ) I'm familiar with the concepts
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- How do you assess yourself on Tactical Design of DDD? (choose one)
+  - ( ) Never used
+  - ( ) I'm familiar with the concepts
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- Which sources have you used to learn about Domain-Driven Design? (Multiple choices)
+  - [ ] Book: Domain-Driven Design by Erik Evans
+  - [ ] Book: Implementing Domain-Driven Design by Vaughn Vernon
+  - [ ] Book: Domain-Driven Design Distilled by Vaughn Vernon
+  - [ ] Book: Learning Domain-Driven Design: Aligning Software Architecture and Business Strategy by Vladik Khononov
+  - [ ] Book: Patterns, Principles, and Practices of Domain-Driven Design by Scott Millett
+  - [ ] Book: Strategic Monoliths and Microservices by Vaughn Vernon
+  - [ ] Conferences
+  - [ ] Blogs and Online Resources
+  - [ ] Workshops and Trainings
+  - [ ] Others... (open text)
+
+### General DDD Concepts
+
+- Which is the most important thing about DDD? It's too open in purpose, there's no right and wrong answer. (Open Text)
+
+- What's a Domain Model? (Open Text)
+
+- What's Domain Modelling? (Open Text)
+
+- In which scenario you would use DDD? (Open Text)
+
+- In which scenario you would NOT use DDD? (Open Text)
+
+- When do you know what you're doing DDD wrong? (Open Text)
+
+- DDD helps you in (Multiple choices)
+  - [ ] Manage Business Complexity
+  - [ ] Mange Technical Complexity
+  - [ ] Define good Microservices
+  - [ ] Have good Architecture because of Tactical Patterns
+  - [ ] Other... (Open text)
+
+- What's a Big Ball of Mud? (Open text)
+
+### Strategic Design
+
+- Which Strategic Patterns are you familiar with? (Multiple choices)
+  - [ ] Bounded Contexts
+  - [ ] Ubiquitous Language
+  - [ ] Core Domain and Sub Domains
+  - [ ] Context Mapping
+  - [ ] Other... (Open text)
+
+- Which of those Patterns are used for the Problem Space? (Multiple choices)
```

Comment:
I would be careful with problem space and solution space. What DDD calls the problem space is more like what product, ux, business people refer to as the strategy and solution space: https://indiyoung.com/about-problem-space/